common/src/main/java/com/wynntils/utils/MathUtils.java

```diff
@@ -4,9 +4,24 @@
  */
 package com.wynntils.utils;
 
+import java.util.HashMap;
 import java.util.Locale;
+import java.util.Map;
 
 public final class MathUtils {
+
+    private static final Map<Integer,Integer> romanNumeralsMap = new HashMap<>();
```

Comment:
TreeMap is better for this?

Reply:
I don't really see why TreeMap would be more appropriate. The map only has 7 items, so memory usage and the order of entries is not an issue. High performance for parsing roman numerals should not be a bottleneck anyway, but AFAIK HashMap is `O(1)`, but TreeMap is `O(log(n))` (for operations relevant here).

Reply:
Sure, fine either way.